README.md

```diff
@@ -2,44 +2,91 @@
 
 [![Coverage Status](https://coveralls.io/repos/github/5afe/safe-gelato-relay-service/badge.svg?branch=main)](https://coveralls.io/github/5afe/safe-gelato-relay-service?branch=main)
 
+**⚠️ Warning: This service is currently going active development. Functionality might change considerably.**
+
+The Safe Gelato Relay Service is a web service which allows relaying transactions via the 
+[Gelato Relay Service](https://docs.gelato.network/developer-services/relay).
+
+This allows for an entity to sponsor on-chain transactions on behalf of the user, allowing them
+to submit Safe transactions without the need to have a wallet with funds. Fee collection can happen via other means
+but it is currently not covered.
+
+## Docker
+
+Development images are currently being published to Docker Hub. 
+You can find them here: https://hub.docker.com/r/safeglobal/safe-gelato-relay-service.
```

Comment:
Nit: UX-wise, [it's better to avoid "here"](https://uxmovement.com/content/why-your-links-should-never-say-click-here/).

```suggestion
Development images are currently being published to [Docker Hub](https://hub.docker.com/r/safeglobal/safe-gelato-relay-service).
```

There are a few other instances of this in the README but I won't mark them all. I'd sooner get your opinion on it first.